src/index.html

```diff
@@ -8,7 +8,7 @@
       href="./styles/main.scss"
```

Comment:
The `href` attribute is pointing to a `.scss` file. Browsers cannot directly interpret SCSS files; they need to be compiled into CSS first. Ensure that the SCSS file is compiled to CSS and link to the resulting CSS file instead.